src/rdbms_vnet/azext_rdbms_vnet/azext_metadata.json

```diff
@@ -1,3 +1,4 @@
 {
-    "azext.minCliCoreVersion": "2.0.24"
+    "azext.minCliCoreVersion": "2.0.24",
+    "azext.maxCliCoreVersion": "2.0.39"
```

Comment:
Does this extension not work on 2.0.40 and higher or is this functionality folded into the CLI in that version?  If the later, then the right thing to do is annotate the CLI to supercede the extension:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#extension-suppression

Reply:
@tjprescott the extension commands have been added to the CLI module

Reply:
@rahulgouthamDOTgs made a PR to the cli repo suppressing the extensions in case a user has already installed the extension: https://github.com/Azure/azure-cli/pull/6694